src/app.ts

```diff
@@ -13,6 +14,14 @@ app.use(logger());
 app.use(cors({
   exposeHeaders: ['Link'],
 }));
+app.use(jsonld({
+  '@language': 'en',
+  '@vocab': 'http://schema.org/',
+  hydra: 'http://www.w3.org/ns/hydra/core#',
+  owl: 'http://www.w3.org/2002/07/owl#',
+  rdf: 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+  rdfs: 'http://www.w3.org/2000/01/rdf-schema#',
+}));
```

Comment:
This isn't ideal, as introduces prefixes that might be unused, but the default might not also be appropriate (eg API documentation has no Schema.org).